src/composables/useConversationInfo.js

```diff
@@ -12,15 +12,20 @@ import { ATTENDEE, CONVERSATION } from '../constants.js'
  * @param {object} payload function payload
  * @param {import('vue').Ref} payload.item conversation item
  * @param {import('vue').Ref} [payload.isSearchResult] whether conversation item appears as search result
- * @param {import('vue').Ref} [payload.exposeMessages] whether to show messages in conversation item
+ * @param {import('vue').Ref} [payload.exposeMessagesRef] whether to show messages in conversation item
+ * @param {import('vue').Ref} [payload.exposeDescriptionRef] whether to show description in conversation item
  */
 export function useConversationInfo({
 	item,
 	isSearchResult = ref(null),
-	exposeMessages = ref(null),
+	exposeMessagesRef = ref(null),
+	exposeDescriptionRef = ref(null),
 }) {
+	const exposeMessages = isSearchResult.value !== null ? !isSearchResult.value : exposeMessagesRef.value
+	const exposeDescription = isSearchResult.value !== null ? isSearchResult.value : exposeDescriptionRef.value
```

Comment:
I'd expect explicitly set arguments to be prioritised
```suggestion
	const exposeMessages = exposeMessagesRef.value !== null ? exposeMessagesRef.value : !isSearchResult.value
	const exposeDescription = exposeDescriptionRef.value !== null ? exposeDescriptionRef.value : isSearchResult.value
```